client/jetpack-cloud/sections/partner-portal/primary/assign-license/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslate } from 'i18n-calypso';
+import { ReactElement, useEffect } from 'react';
+import CardHeading from 'calypso/components/card-heading';
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import AssignLicenseForm from 'calypso/jetpack-cloud/sections/partner-portal/assign-license-form';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+
+export default function AssignLicense( { sites }: any ): ReactElement {
+	const translate = useTranslate();
+
+	useEffect( () => {
+		const layoutClass = 'layout__content--partner-portal-assign-license';
+		const content = document.getElementById( 'content' );
+
+		if ( content ) {
+			content.classList.add( layoutClass );
+
+			return () => content.classList.remove( layoutClass );
+		}
+	}, [] );
```

Comment:
We should move this to a hook so we don't keep duplicating it for other similar layouts.

Reply:
Agreed. As discussed above, we need to do a short sprint to clean up some of these patterns. For now, this isn't a blocker IMO.